{
  "public_data_at_usn": "0001-01-01T00:00:00Z",
  "crd": "0001-01-01T00:00:00Z",
  "candidate": "CVE-2020-29657",
  "public_date": "2020-12-09T09:15:00Z",
  "references": [
    {
      "reference": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-29657"
    },
    {
      "reference": "https://github.com/jerryscript-project/jerryscript/issues/4244"
    }
  ],
  "description": "In JerryScript 2.3.0, there is an out-of-bounds read in main_print_unhandled_exception in the main-utils.c file.",
  "ubuntu_description": "",
  "notes": [],
  "bugs": [
    {
      "bug": "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=977736"
    }
  ],
  "priority": "medium",
  "discovered_by": "",
  "assigned_to": "",
  "patches": [
    {
      "package_name": "iotjs",
      "release_patches": [
        {
          "release_name": "hirsute",
          "status": "needs-triage",
          "note": ""
        }
      ]
    }
  ],
  "upstreams": []
}
